### PR TITLE
Progress for upgrading Mailer call to send mail to author and team

### PR DESCRIPTION
**Pull request by autumnfound**
Includes Qute templates, Mailer extension settings, and logic to update
form once finished.

### CODE REVIEW

**Comment by autumnfound** [comment]
Build failed previously due to timeout in yarn build. Retriggering to see if we can get a clean build that actually hits the changes.

File: src/main/java/org/eclipsefoundation/react/service/impl/DefaultMailerService.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        // get the user object for the current form
+        List<EclipseUser> users = accounts.getUsers(form.getUserID(), null, null);
+        if (users == null || users.isEmpty()) {
+            throw new IllegalStateException("Could not find user with ID " + form.getUserID());
+        }
+        EclipseUser user = users.get(0);
+        // TODO use this for mail TO field rather than webdev
+        user.getMail();*/
+        // send the email, using the users primary email address
+        Mail m = Mail.withHtml("webdev@eclipse-foundation.org", "Thank you for completing the member enrollment form",
```
Comment (mbarbero) [inline src/main/java/org/eclipsefoundation/react/service/impl/DefaultMailerService.java:72]: I guess this is still a WIP, reason why address is still `webdev@eclipse-foundation.org`, correct?

Comment (autumnfound) [reply src/main/java/org/eclipsefoundation/react/service/impl/DefaultMailerService.java:72]: Correct. We have an issue with our OAuth server in drupal that causes the accounts call to fail when using the quarkus builtin stuff. We've got an issue to track fixing it, but I've got the real code roughed in above. Once its ready, I'll uncomment and replace the hard coded email

**Review by mbarbero — APPROVED** [review]

**Comment by autumnfound** [comment]
@mbarbero re-requested as I've updated a few fields. We use BCC now, I've disabled a test that was breaking and couldn't be easily fixed w/ this version of Quarkus (PR made for upgrade here https://gitlab.eclipse.org/eclipsefdn/webdev/eclipsefdn-api-common/-/merge_requests/13)

**Review by mbarbero — APPROVED** [review]

**Pull request merged**